csharp/src/Ice/Connection.cs

```diff
@@ -545,8 +545,8 @@ private async ValueTask AcceptStreamAsync()
                 {
                     if (stream.IsBidirectional)
                     {
-                        var exception = new ObjectNotExistException();
-                        response = new OutgoingResponseFrame(request, exception);
+                        response = new OutgoingResponseFrame(request,
+                                                             new ObjectNotExistException(RetryPolicy.OtherReplica));
```

Comment:
Should `ONE` have a baked in default of` RetryPolicy.OtherReplica` which can be overridden in the constructor?

Reply:
No: I find that an automatic retry policy for any exception would be error prone: the natural default is for an exception to NOT trigger any retry.
If you throw ONE in your application code and want retry behavior, you need to specify the retry policy explicitly - if you don't, you get NoRetry.

Reply:
I agree NoRetry being the default for all exceptions makes it easier to understand.